configure.py

```diff
@@ -51,6 +51,7 @@
 valid_mips_fpu = ('fp32', 'fp64', 'fpxx')
 valid_mips_float_abi = ('soft', 'hard')
 valid_intl_modes = ('none', 'small-icu', 'full-icu', 'system-icu')
+minimum_icu = 57
```

Comment:
Could you read this with:
https://github.com/nodejs/node/blob/bce91be01e333aa67874c89eeea419eb68dcc33a/configure.py#L1266-L1267

Reply:
@refack hm. Doesn't make sense with the format of current_ver.dep … the minimum ICU is global, doesn't need to be repeated multiple times.

Reply:
@refack I split the minimum version to a separate file.

Reply:
We could change the format of `tools/icu/current_ver.dep` to be:
```json
{
  "current_minimum": 57,
  "packages": [ ... current content ... ]
}